### ISSUE TITLE
ClassNotFoundException: androidx.dynamicanimation.R when androidx.dynamicanimation:dynamicanimation is transitively included in runtime classpath

### ISSUE THREAD

**Issue by elihart** [cashapp/paparazzi#418]
On the latest 1.0 snapshot I have a library module where `testDebugUnitTest` task fails with this crash
```
java.lang.ClassNotFoundException: androidx.dynamicanimation.R
	at java.base/jdk.internal.loader.BuiltinClassLoader.loadClass(BuiltinClassLoader.java:581)
	at java.base/jdk.internal.loader.ClassLoaders$AppClassLoader.loadClass(ClassLoaders.java:178)
	at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:522)
	at java.base/java.lang.Class.forName0(Native Method)
	at java.base/java.lang.Class.forName(Class.java:315)
	at app.cash.paparazzi.internal.PaparazziCallback.initResources(PaparazziCallback.kt:62)
	at app.cash.paparazzi.Paparazzi.prepare(Paparazzi.kt:146)
	at app.cash.paparazzi.Paparazzi$apply$statement$1.evaluate(Paparazzi.kt:121)
```

The gradle build scan shows that a 3rd party library I have an `implementation` dependency on in turn has a dependency on `com.google.android.material:material` which then brings in `androidx.dynamicanimation:dynamicanimation`, so that it is on the `debugRuntimeClasspath`. 

It seems like the paparazzi plugin includes the resource file from the dynamicanimation library even though it is not on my debug test runtime classpath, so this crash happens.

I can work around it by adding an explicit testImplementation dependency on androidx.dynamicanimation:dynamicanimation, but hopefully this can be better addressed in the plugin itself.

**Comment by jrodbx** [comment]
@elihart any updates here?  has a more recent version fixed this?  if not, could you provide a small repro/example to help triage faster?

**Comment by elihart** [comment]
I just checked and I no longer see the issue after removing the workaround I used, so it seems like the issue was fixed at some point (we’re on 1.2)
